src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -528,6 +529,9 @@ static irqreturn_t sched_exec_isr(int irq, void *arg)
 		return IRQ_NONE;
 	}
 
+	if(zocl_cu_get_control(&zdev->exec->zcu[cu_idx]) == AP_CTRL_CHAIN)
```

Comment:
You may want to add a space between if and (. Can you please run XRT/build/checkpatch to make sure we do not introduce any new warning?